README.md

```diff
@@ -31,12 +31,22 @@ const AVN_GATEWAY_URL = 'https://...';
 // The AvN address of the relayer you will be using, as supplied by Aventus:
 const AVN_RELAYER = '5Ekag...';
 
+// The AvN address of the payer you will be using:
+const PAYER = '5G7B3...';
+
 // The Ethereum address of the Authority required for minting NFTs, as supplied by Aventus:
 const AVN_AUTHORITY = '0xD3372...';
 
 async function main() {
   const options = { suri: '0x816ef9f2c7f9e8c013fd5fca220a1bf23ff2f3b268f8bcd94d4b5df96534173f'};
+  // For split fee functionality we must specify the payer in the options object.
+  const splitFeeOptions = { suri: '0x816ef9f2c7f9e8c013fd5fca220a1bf23ff2f3b268f8bcd94d4b5df96534173f', payerAddress: PAYER };
```

Comment:
I think we should leave the `payerAddress` as null or omit it to show that its optional.

Reply:
That is what I am trying to show on line 45: `const defaultSplitFeeOptions = { suri: '0x816ef9f2c7f9e8c013fd5fca220a1bf23ff2f3b268f8bcd94d4b5df96534173f', hasPayer: true };`

Should I change it?

Reply:
I see, I added a small change to the comments but we can leave this as is.